### DISCUSSION TITLE
zzz驱动盘晒非楼（神盘镇楼）

### DISCUSSION THREAD

**Discussion by wthout-i** [share121/inter-knot#5244] [绝区零]
![Image_1726833134485](https://github.com/user-attachments/assets/39c213f9-2fbc-4cd6-80b3-35e3cd4796c9)
驱动盘非欧相对于抽卡并不会引起如此大的情绪波动，所以大家能更加没有顾虑的看乐子（雾）但是驱动盘由于其特殊的性质，某种程度上比抽卡还要看脸，故开此楼，望遇见更加逆天的词条

**Comment by Kazuha-hhh** [comment]
看开点，至少这个驱动盘挺耐摔啊对吧

**Comment by Doubudou45** [comment]
![17268968791993fa5b9809ebd5404274f4830e3b70840f022e72a446cb391615b7997444ba088 0](https://github.com/user-attachments/assets/cefdbc33-2814-4737-9c4d-98a8e0ad4306)
是时候拿出这个了！

**Comment by Nie7bai** [comment]
![？](https://github.com/user-attachments/assets/22626e46-e10d-4c78-9220-79fb6614c227)
随一盘<img src="https://bbs-static.miyoushe.com/static/2023/05/22/a2fa805a0a782cdb8ab44ddaf8d5343d_9213146029750742963.png" alt="阿姬倒地了02" width="75" >
